packages/context-entries/lib/Keys.ts

```diff
@@ -298,3 +301,18 @@ export const KeysRdfJoin = {
    */
   lastPhysicalJoin: new ActionContextKey<string>('@comunica/bus-rdf-join:lastPhysicalJoin'),
 };
+
+export const KeysTrackableStatistics = {
```

Comment:
Perhaps this can now just become `KeysStatistics` ?

Reply:
Agreed, this should be done now